modules/programs/vim.nix

```diff
@@ -19,7 +19,16 @@ in
         default = null;
         description = "Whether to show line numbers.";
       };
-
+      relativeNumber = mkOption {
+        type = types.nullOr types.bool;
+        default = null;
+        description = "Whether to show relative line numbers column.";
+      };
```

Comment:
Option declarations should be separated by empty line.

Reply:
Can I overwrite my history to fix this without additional commit in the repo, so history would be clean?

Reply:
Yes, you need to do an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_changing_multiple). It's a bit involved so I won't attempt to describe it fully but basically you want to 

1. start an interactive rebase of the last two commits,

2. choose to "edit" them both,

3. when git stops at each of the two commits edit the `vim.nix` file so that it now has the empty line,

4. tell git to "amend" the stopped commit with the changes you made to `vim.nix`,

5. tell git to continue, and finally

6. force push your branch to github.

Note, it is generally OK to force push to PR branches since they are completely under your control and it is only you who work on the branch. But terrible _terrible_ things happen if you force push to a branch where other people work!

Reply:
Thanks for help @rycee! I think I've fixed capitalization and coding style without any new commits.